chronix-server-test-integration/src/main/java/de/qaware/chronix/solr/CSVImporter.java

```diff
@@ -43,7 +42,7 @@
  *
  * @author f.lautenschlager
  */
-public class CSVImporter {
+public final class CSVImporter {
```

Comment:
\[Checkstyle\] ERROR: Utility classes should not have a public or default constructor\.